pathod/language/http.py

```diff
@@ -171,17 +171,17 @@ def reason(self):
         return self.tok(Reason)
 
     def preamble(self, settings):
-        l = [self.version, " "]
+        l = [self.version, b" "]
         l.extend(self.status_code.values(settings))
         status_code = int(self.status_code.value)
-        l.append(" ")
+        l.append(b" ")
         if self.reason:
             l.extend(self.reason.values(settings))
         else:
             l.append(
                 status_codes.RESPONSES.get(
                     status_code,
-                    "Unknown code"
+                    b"Unknown code"
                 )
```

Comment:
I've done that here: https://github.com/mitmproxy/mitmproxy/pull/1210/commits/b3763fd850f13ed8616c454910d29d00c371cc65